### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: docs/HowItWorks.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ foreach ($fileInfos as $fileInfo) {
 
 ### 2.2 Rectify Phase
 
-- When all nodes are ready, applicies iterates all active Rector
-- Each nodes is compared to `$rector->getNodeTypes()` method to see, if this Rector should do some work on it, e.g. is this class name called `OldClassName`?
+- When all nodes are ready, applicies iterates all active Rectors
```
Comment (staabm) [inline docs/HowItWorks.md]: Not sure what `applicies` means, or is it a typo?

Comment (TomasVotruba) [reply docs/HowItWorks.md]: I think it should be `application`

Comment (staabm) [reply docs/HowItWorks.md]: Thx, fixed

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**